### PR TITLE
install bee using install.sh

### PR DESCRIPTION
**Pull request by vandot**
`install.sh` just got [merged](https://github.com/ethersphere/bee/pull/468) into bee master, this script will detect OS and ARCH and install/upgrade bee binary to proper location

### CODE REVIEW

File: content/bee-docs/installation/index.md [hunk 1]
```diff
@@ -12,14 +12,15 @@ The client can be installed on MacOS and various Linux flavors.
...
 At every official release, we release the binary files of our software. You can download them by navigating to [releases](https://github.com/ethersphere/bee/releases).
 
-A convenient way to download the binary is:
+A convenient way to download the binary is using `install.sh` script:
 
-```sh
-wget https://github.com/ethersphere/bee/releases/latest/download/bee-darwin-amd64 -O /usr/local/bin/bee
-chmod +x /usr/local/bin/bee
-```
+Grab the latest release:
```
Comment (Eknir) [inline content/bee-docs/installation/index.md:17]: A comment:

The docs are made for the latest release (TAG). You are mentioning "latest release" here as `master`, whereas in fact you mean `edge`. 

Could you please correct that, and additionally also put emphasis on downloading the latest TAG (i.e. latest release). Suggesting to change the order (put downloading latest release (TAG) before instructions to download edge) and perhaps put a warning message for downloading the latest master (as the docs might not be up-to-date).

**Review by Eknir — CHANGES_REQUESTED** [review]
Thanks for this PR!

I believe you are using the wrong terminology (calling master "latest release" and the latest release "TAG"). Additionally, the docs are made for the latest release, so instructions to download the edge should come with a warning.

**Comment by Eknir** [comment]
@significance , heads up

File: content/bee-docs/installation/index.md (hunk 1 above)
Comment (vandot) [reply content/bee-docs/installation/index.md:17]: @Eknir put a warning message where? when somebody executes install using script if its `edge` to show confirmation dialog? or just warn message?

Comment (Eknir) [reply content/bee-docs/installation/index.md:17]: I don't think it is needed to put this warning message in the script itself. I meant more a message in the documentation, saying something along the lines that if the user installs the edge (master) version, then the documentation is not up-to-date with the latest changes. Additionally, you could mention that the network to which the nodes connects by default runs the latest release, so even the node might behave inconsistent.

**Comment by Eknir** [comment]
@vandot , I just made the changes (also because we have a new docs in general). https://github.com/ethersphere/docs.github.io/commit/361033af5c4ddab800ba12893b635785df27f06b

Let me know if it is good. Sorry, no PR. Pushed to master accidentally

**Comment by Eknir** [comment]
If it is good, could you close this issue?

**Pull request closed without merging**